src/Menu/Menu.js

```diff
@@ -76,11 +75,7 @@ type Props = DefaultProps & {
   transitionDuration?: number | 'auto',
 };
 
-export const styleSheet = createStyleSheet('MuiMenu', {
-  root: {
-    maxHeight: 250,
-  },
-});
+export const styleSheet = createStyleSheet('MuiMenu', {});
```

Comment:
I don't think that we need any CSS customization power on that component. Let me know :).